anagramGroups.js

```diff
@@ -0,0 +1,61 @@
+let strs = ['eat', 'tea', 'tan', 'ate', 'nat', 'bat'];
+
+let group = new Array();
```

Comment:
Why not simply `let group = []` ?

Reply:
Agree! Probably it's due to my still low practice level. Fixed.